### ISSUE TITLE
feat: validation for dapr component name

### ISSUE THREAD

**Issue by kewalaka** [Azure/terraform-azurerm-avm-res-app-managedenvironment#25]
Need to add a variable condition to check for this:

> "Invalid Dapr Component name 'my_dapr_component'. A name must consist of lower case alphanumeric characters, '-' or '.', start with an alphabetic character, and end with an alphanumeric character and cannot have '--'. The length must not be more than 60 characters."

**Comment by segraef** [comment]
@kewalaka is this sill open?

**Comment by kewalaka** [comment]
yep, haven't had a burning need for it.  Just tripped me up in a CI run.  When I next do some work on the module I'll address, unless someone gets there first, no plans made yet.

**Comment by kewalaka** [comment]
fixed in #161 / released
